sklearn/tests/test_dummy.py

```diff
@@ -534,14 +534,26 @@ def test_constant_strategy_multioutput():
     _check_predict_proba(clf, X, y)
 
 
-def test_constant_strategy_exceptions():
+@pytest.mark.parametrize('y, params, err_msg', [
+    ([2, 1, 2, 2],
+     {'random_state': 0},
+     "Constant.*has to be specified"),
+    ([2, 1, 2, 2],
+     {'constant': [2, 0]},
+     "Constant.*should have shape"),
+    (np.transpose([[2, 1, 2, 2], [2, 1, 2, 2]]),
```

Comment:
You are missing the following case

```python
(np.transpose([[2, 1, 2, 2], [2, 1, 2, 2]]),
 {'constant': [2, 'unknown']},
 "constant=my-constant.*Possible values.*[1, 2]")
```
This would be the `multi-output` case